src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,76 @@
+package calculator;
+
+public class Calculator {
+
+    public static final String DELIMITER = ",|:";
+    public static int cal(String text)
+    {
+        if(isBlank(text))
+        {
+            return 0;
+        }
+        return sum(toInts(split(text)));
+    }
+
+    private static String[] split(String text)
+    {
+        if(isCustom(text))
+        {
+            return customSplit(text);
+        }
+        return text.split(DELIMITER);
+    }
+
+    private static boolean isCustom(String text)
+    {
+        if(text.split("//|\n").length == 3)
+        {
+            return true;
+        }
+        return false;
+    }
+
+    private static String[] customSplit(String text)
+    {
+        String[] textList = text.split("//|\n");
+        return textList[2].split(textList[1]);
+    }
+
+    private static boolean isBlank(String text) {
+        return text == null || text.isBlank();
+    }
+
+    private static int sum(int[] numbers)
+    {
+        includeMinus(numbers);
+        int sum = 0;
+        for(int number: numbers)
+        {
+            sum += number;
+        }
+        return sum;
+    }
+
+    private static void includeMinus(int[] numbers)
+    {
+        try {
+            for (int i : numbers)
+            {
+                System.out.println(i);
+                if(i < 0) throw new RuntimeException("DO NOT INPUT MINUS NUMBER");
+            }
+        } catch (RuntimeException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
```suggestion
    private static void includeMinus(int[] numbers)
    {
        for (int number : numbers)
        {
            validatePositive(number);
        }
    }

    private static void validatePositive(int i) {
        if(i < 0) {
            throw new IllegalArgumentException("DO NOT INPUT MINUS NUMBER");
        }
    }
```

이렇게 변경할 수 있을 것 같아요

추가로 RuntimeException 보다는 더 적절한 표준예외인 IllegalArgumentException이 알맞을 것 같습니다!

Reply:
`System.out.println(i);` 디버깅을 위한 코드 같네요 
이런 부분들은 디버깅 모드를 사용하는 방법으로 변경하는 것을 추천드려요

Reply:
i라는 변수명 보다는 의미 전달이 잘되는 이름으로 변경하는 게 좋을 것 같네요
변수명에는 항상 의도가 나타나야합니다!

Reply:
네 저렇게 디버깅하는게 습관이 었는데 디버깅 모드를 사용하도록 노력하겠습니다 !
또한 변수, 메소드 등 이름 설정 조금 더 신경쓰겠습니다!